### PR TITLE
Shenghung kuan 43 create theme for app

### LINKED ISSUE #43: Create Theme for app
We want to have all styling be unified and standardized; we can take a look at themes in MUI to execute this or explore other options:
https://mui.com/material-ui/customization/theme-components/

### PR DESCRIPTION
**Pull request by shenghungKuan**
close #43

### CODE REVIEW

**Review by lawrenceslng — CHANGES_REQUESTED** [review]
I think the code looks pretty good, I think you can open an issue for the following 2 recommendations and we can come back to it when we have time:

1) there are still some mobile responsiveness code in some files, would be nice to consolidate within `Theme.js`
2) there are still some stylings for things like margins and stuff that could be consolidated too

Can you pull latest dev branch and merge with your branch and update the PR? Then I'll run your branch and check if there's any styling that we should fix

**Comment by shenghungKuan** [comment]
Since some of them have different responsive sizes and margins, I just left it as what it is.
If we want to consolidate all of them, I may need your opinions or to discuss which one to apply.
And yes, I'll merge the dev into this branch first.

File: client/src/components/Root/Banner.js
Comment (lawrenceslng) [inline client/src/components/Root/Banner.js:1]: we need to add a little more margins between the three parts within the Banner and the 2nd part needs a heavier font weight:

latest:
image


Wireframe:
image

**Review by lawrenceslng — CHANGES_REQUESTED** [review]
Just 1 little change

File: client/src/components/Root/Banner.js
Comment (shenghungKuan) [reply client/src/components/Root/Banner.js:1]: Good catch!!!

Comment (shenghungKuan) [reply client/src/components/Root/Banner.js:1]: 截圖 2023-07-20 下午12 38 32
截圖 2023-07-20 下午12 38 39

Comment (shenghungKuan) [reply client/src/components/Root/Banner.js:1]: @lawrenceslng @zhunan123 please take a look and approve if it looks good to you. Thanks.

**Review by lawrenceslng — APPROVED** [review]
Approved

**Pull request merged**